app/controllers/users/sessions_controller.rb

```diff
@@ -1,65 +1,3 @@
 # frozen_string_literal: true
 
-class Users::SessionsController < Devise::SessionsController
-  class EmailNotFoundError < StandardError; end
-  class LoginIncompleteError < StandardError; end
-
-  TEST_USERS = %w[
-    admin@example.com
-    ambition-institute-early-career-teacher@example.com
-    education-development-trust-early-career-teacher@example.com
-    teach-first-early-career-teacher@example.com
-    ucl-early-career-teacher@example.com
-    mentor@example.com
-  ].freeze
-
-  before_action :mock_login, only: :create, unless: -> { Rails.env.production? }
-  before_action :redirect_to_dashboard, only: %i[sign_in_with_token redirect_from_magic_link]
-  before_action :ensure_login_token_valid, only: %i[sign_in_with_token redirect_from_magic_link]
-
-  def create
-    super
-  rescue LoginIncompleteError
-    render :login_email_sent
-  rescue EmailNotFoundError
-    render :email_not_found
```

Comment:
Those templates can go too. Also, I think cypress test is failing cause it tries to see text on one of them - time to update it :)